fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/control/PopulateParameterNames.java

```diff
@@ -52,15 +54,27 @@ public PopulateParameterNames(String adminSchemaName, String schemaName, Integer
     public void run(IDatabaseTranslator translator, Connection c) {
         final String stmtVariable = String.format("SET %s.SV_TENANT_ID = %d", adminSchemaName, tenantId);
         final String stmtResourceTypeInsert;
+        final String PARAMETER_NAMES = String.format("SELECT PARAMETER_NAME_ID, PARAMETER_NAME FROM %s.parameter_names", schemaName);
         if (tenantId != null) {
             stmtResourceTypeInsert =
                     String.format("INSERT INTO %s.parameter_names (MT_ID, PARAMETER_NAME_ID, PARAMETER_NAME) "
                             + "VALUES (%s.sv_tenant_id, ?, ?)", schemaName, adminSchemaName);
         } else {
             stmtResourceTypeInsert =
-                    String.format(
-                            "INSERT INTO %s.parameter_names (PARAMETER_NAME_ID, PARAMETER_NAME) " + "VALUES (?, ?)",
-                            schemaName);
+                    String.format("INSERT INTO %s.parameter_names (PARAMETER_NAME_ID, PARAMETER_NAME) " + "VALUES (?, ?)", schemaName);
+        }
+
+        Map<String, Integer> values = new HashMap<>();
```

Comment:
As this is being run as maintenance task I supposed we're not going to be too worried about concurrency. Selecting first then inserting what's missing isn't thread-safe but it's probably not a big deal here.

Reply:
Good point, I'll have to defer this part... if customers hit we can add USING RS

Reply:
Yeah, I'm not too worried about this in this context.